clients/runtime/src/lib.rs

```diff
@@ -79,7 +77,7 @@ impl Config for SpacewalkRuntime {
     type Hash = H256;
     type Hashing = BlakeTwo256;
     type AccountId = AccountId;
-    type Address = Self::AccountId;
+    type Address = Address;
```

Comment:
This is the elementary fix that ensures that subxt uses the actual address format used in the (testchain) runtime and not the account id format.